src/react-apps/applications/ux-editor/src/utils/render.tsx

```diff
@@ -30,13 +30,16 @@ export function renderSelectDataModelBinding(
   selectedElement: any,
   onDataModelChange: any,
   language: any,
+  label?: string,
+  returnValue?: any,
 ): JSX.Element {
   return (
     <div>
-      {renderPropertyLabel(language.ux_editor.modal_properties_data_model_helper)}
+      {renderPropertyLabel(label ? label : language.ux_editor.modal_properties_data_model_helper)}
```

Comment:
Maybe a combination of these to make it more understandable?

Reply:
Agreed, now looks like "Lenke til datamodell for X"